### PR TITLE
Heperparameterisation of Transaction Classification

### PR DESCRIPTION
**Pull request by Samruddhi-ds**
1. Use hyper parameterisation to identify better combinations of dropout and dense layer configurations.
2. Setup tensorflow to log the score for each hyperparameter combination.
3. Generate code that will save the best performing model from the validation score.

### CODE REVIEW

**Review by PradiptaDutta63 — CHANGES_REQUESTED** [review]
Please upload a python script for the frontend to use it.

**Comment by Samruddhi-ds** [comment]
Hi Pradipta, thank you for your review. 
I have created a new PR recently; I have retrained this model and submitted a py file for the frontend to use the latest version. 

Thank you.
